python/python_frontend/python_bindings.cpp

```diff
@@ -3720,7 +3722,14 @@ void initNvFuserPythonBindings(PyObject* module) {
          Tensor offsets,
          Tensor scale1,
          Tensor scale2,
-         std::optional<PrimDataType> dtype) -> Tensor {
+         std::optional<Tensor> alpha,
+         std::optional<Tensor> bias,
+         std::optional<Tensor> beta,
```

Comment:
Again, `optional` seems useless because `Tensor` already has a "null" state.

Reply:
This is `nvfuser::python_frontend::Tensor` so we don't have a null state on this.
https://github.com/NVIDIA/Fuser/blob/929cc4c753a44ea792356b5d49f3aa923b50799d/python/python_frontend/fusion_definition.h#L41-L76